motorlib/grain.py

```diff
@@ -103,17 +107,20 @@ class PerforatedGrain(Grain):
     geomName = 'perfGrain'
     def __init__(self):
         super().__init__()
-        self.props['inhibitedEnds'] = EnumProperty('Inhibited ends', ['Neither', 'Top', 'Bottom', 'Both'])
+        self.props['inhibitedEnds'] = EnumProperty('Inhibited ends', [InhibitedEnds.NEITHER,
+                                                                      InhibitedEnds.TOP,
+                                                                      InhibitedEnds.BOTTOM,
+                                                                      InhibitedEnds.BOTH])
         self.wallWeb = 0 # Max distance from the core to the wall
 
     def getEndPositions(self, regDist):
-        if self.props['inhibitedEnds'].getValue() == 'Neither': # Neither
+        if self.props['inhibitedEnds'].getValue() == InhibitedEnds.NEITHER: # Neither
```

Comment:
Nitpick: The comments here are probably even less important now!

Reply:
Good catch!